js/core/finances.js

```diff
@@ -31,12 +35,33 @@ define(["dao", "globals", "lib/underscore"], function (dao, g, _) {
                         t.seasons[s].expenses.minTax.amount = g.minPayroll - payrolls[t.tid];
                         t.seasons[s].cash -= t.seasons[s].expenses.minTax.amount;
                     } else if (payrolls[t.tid] > g.luxuryPayroll) {
-                        t.seasons[s].expenses.luxuryTax.amount = g.luxuryTax * (payrolls[t.tid] - g.luxuryPayroll);
+                        amount = g.luxuryTax * (payrolls[t.tid] - g.luxuryPayroll);
+                        collectedTax += amount;
+                        t.seasons[s].expenses.luxuryTax.amount = amount;
                         t.seasons[s].cash -= t.seasons[s].expenses.luxuryTax.amount;
                     }
 
                     return t;
                 }
+            }).then(function() {
+                var payteams;
+                payteams = payrolls.filter(function(x) { return x <= g.salaryCap; });
+                if (payteams.length > 0) {
```

Comment:
Might as well also check if `collectedTax > 0` to save running some code that does nothing.